### ISSUE TITLE
no entry found for key

### ISSUE THREAD

**Issue by musicformellons** [t-rex-tileserver/t-rex#60]
I get this:
```
root@vps:/home/usr# RUST_BACKTRACE=1 t_rex generate --extent=538432.3009181806,6851282.3420069255,538432.3009181806,6851282.3420069255 --minzoom=11 --maxzoom=21 --overwrite=true --config /home/usr/osm.toml
2017-09-18 11:36:59.178 INFO Reading configuration from '/home/usr/osm.toml'
2017-09-18 11:36:59.214 INFO Tile cache directory: /mnt/vdb/tmp/mvtcache
Generating tileset 'datapoints'...
Level 21: 0 / 1 [-------------------------------------------------------------------------------------------------------------------------------------------] thread 'main' panicked at 'no entry found for key', /checkout/src/libcore/option.rs:794
stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: main
  11: __libc_start_main
  12: <unknown>

```

**Comment by pka** [comment]
Which version are you using? Since 0.7.4 extents are in WGS84:
https://github.com/t-rex-tileserver/t-rex/blob/master/CHANGELOG.md#breaking-changes-2

**Comment by musicformellons** [comment]
I am on 0.7.5, maybe I just upgrade and take it from there?!

**Comment by musicformellons** [comment]
Actually, it seems that the tiles are updated nicely, so except from the errormessage itself, it is working as should I think.

**Comment by pka** [comment]
Then I think it's one the bugs fixed in 0.7.6 and 0.7.7. I recommend updating to 0.7.8.
